.pylintrc

```diff
@@ -0,0 +1,243 @@
+[MASTER]
```

Comment:
How much of this `.pylintrc` file is "stock" versus customized? Assuming `.pylintrc` files just override the built-in defaults, I think we should trim this down only to project-specific overrides.

Reply:
That's correct. This is already a fairly cut-down version, I've kept some things that I figured would be nice to expose in case we ever want to tweak them, but if you'd like I can trim this down just to the customized values.

Reply:
Went ahead and cut it down just to the customized options.